app/src/main/java/net/gsantner/markor/format/todotxt/TodoTxtTextActions.java

```diff
@@ -298,7 +304,7 @@ public boolean onLongClick(View v) {
                     return true;
                 }
                 case R.string.tmaid_todotxt_current_date: {
-                    _hlEditor.getText().insert(origSelectionStart, " due:" + SttCommander.getDaysFromToday(3));
+                    setKeyDate(origTask, "t", 3, getContext().getString(R.string.set_threshold_date));
```

Comment:
t?

Reply:
The t is for `t:yyyy-mm-dd` threshold dates. After thinking about this some more, I feel that we do not need threshold dates.

I have updated the pr to:
1. Short press of date inserts date (if a date is selected, the date is replaced)
2. Long press inserts or updates a due date